common/src/main/java/org/fao/geonet/utils/Xml.java

```diff
@@ -717,6 +732,24 @@ public static String getString(Document data) {
         return outputter.outputString(data);
     }
 
+    /**
+     *
+     * @param data
+     * @return
+     */
+    public static String getString(Node data) {
+        try {
+            TransformerFactory tf = TransformerFactory.newInstance();
```

Comment:
I think it isn't a good idea from the point of view of the performance to create a new `TransformerFactory` every time the method is called. Shouldn't we use `TransformerFactoryFactory.getTransformerFactory()` instead that uses a singleton?
```suggestion
            TransformerFactory tf = TransformerFactoryFactory.getTransformerFactory();
```